.actions/ipynb-render.sh

```diff
@@ -40,6 +49,11 @@ else
 fi
 
 cp $meta_file $pub_meta_file
-git add ".notebooks/$1.yaml"
+git add $pub_meta_file
+
+if [ ! -z $thumb_file ]; then
+  cp $thumb_file $pub_thumb_file
```

Comment:
isn't the copy already in the CLI with `copyfile(old_path_thumb, new_thumb)`

Reply:
This copies from original location to `.notebooks`, then CLI copies from `.notebooks` to `docs/source/_static/images`